myia/compile/backends/pytorch.py

```diff
@@ -0,0 +1,191 @@
+"""Linear implementation using pytorch."""
+
+import torch
+import torch.utils.dlpack
+
+from . import Backend
+from ..transform import CompileGraphs, nonlinear_ops
+
+from ...dtype import Int, UInt, Float, Bool
+from ...prim import Primitive, ops as P
+
+
+_type_map = {
+    Int[8]: torch.int8,
+    Int[16]: torch.int16,
+    Int[32]: torch.int32,
+    Int[64]: torch.int64,
+    UInt[8]: torch.uint8,
+    Float[16]: torch.float16,
+    Float[32]: torch.float32,
+    Float[64]: torch.float64,
+    Bool: torch.uint8,
+    # This is a hack but we really need uint64 support
+    UInt[64]: torch.int64,
+}
+
+
+def type_to_pytorch_type(t):
+    """Map myia types to pytorch types."""
+    if t not in _type_map:  # pragma: no cover
+        raise TypeError(f"Unsupported type: {t}")
+    return _type_map[t]
+
+
+simple_mapping = {
+    P.scalar_add: lambda a, b: a + b,
```

Comment:
Could we just use `torch.add` instead? Ditto for the rest of the mappings.

Reply:
`torch.add` doesn't actually add two tensors, it just add a scalar to a tensor.  Same for the rest.